CHANGELOG.md

```diff
@@ -9,9 +9,13 @@ All notable changes to the Pony compiler and standard library will be documented
 
 ### Added
 
+- Version command to print version
+
+- New command for creating empty changelogs
 
 ### Changed
 
+- The changelog file name has become an optional argument
```

Comment:
This seems to be a breaking change. "optional" doesn't quite capture the level of change this seems to be. can you verify that is correct?